vite.config.mts

```diff
@@ -97,7 +99,9 @@ export default defineConfig(({ mode }) => {
         Object.entries(publicFilesEnvVars).map(([key, value]) => [`import.meta.env.${key}`, value]),
       ),
       ...Object.fromEntries(
-        Object.entries(publicFilesEnvVars).map(([key, value]) => [`process.env.${key}`, value]),
+        Object.entries(env)
+          .filter(([key]) => key.startsWith('VITE_'))
+          .map(([key, value]) => [`process.env.${key}`, JSON.stringify(value)]),
```

Comment:
Only add prefixed variables to our process.env, manually so build:packages is happy, previously we were overwridding the whole `process.env` object which was creating bugs when trying to access private env variables